src/components/Input/Input.module.css

```diff
@@ -0,0 +1,19 @@
+.input_input {
+  padding: 4px;
+
+  height: 34px;
+  border: solid 2px var(--color-font-secondary);
+  border-radius: 4px;
+
+  color: var(--color-font-secondary);
```

Comment:
Entered text looks almost like placeholder. Make it more visible (darker)

<img width="372" alt="image" src="https://user-images.githubusercontent.com/23063254/212468864-c249cb5e-5aa2-4164-9348-53a55889effe.png">